src/meta/src/stream/stream_graph.rs

```diff
@@ -524,31 +534,10 @@ impl StreamGraphBuilder {
     /// ids if it is a `ChainNode`.
     fn build_inner(
         &self,
-        ctx: &mut CreateStreamingJobContext,
         stream_node: &StreamNode,
-        fragment_id: GlobalFragmentId,
         upstream_actor_id: &mut HashMap<u64, OrderedActorLink>,
         upstream_fragment_id: &mut HashMap<u64, GlobalFragmentId>,
     ) -> MetaResult<StreamNode> {
```

Comment:
The comments of this method can by the way be refactored as well.

Reply:
This doc is not updated when the logic of internal tables is mixed into this function. So after we move it out, the current doc exactly reflects what it does and there's no need to update it. 😂